
class TopException(Exception):

    def __init__(self):
        pass